### PR TITLE
[MRG]: Fixed simulations dropdown list adding loaded data

### PR DESCRIPTION
**Pull request by kmilo9999**
The simulations dropdown list was adding loaded data, and hnn-gui cant export non-simulated data.

solves #824

### CODE REVIEW

File: hnn_core/gui/gui.py [hunk 1]
```diff
@@ -1726,10 +1726,11 @@ def run_button_clicked(widget_simulation_name, log_out, drive_widgets,
             simulation_status_bar.value = simulation_status_contents[
                 'finished']
 
-            sim_names = [sim_name for sim_name
-                         in simulation_data]
-            simulations_list_widget.options = sim_names
-            simulations_list_widget.value = sim_names[0]
+            run_simulations = [sim_name for sim_name in simulation_data
+                               if simulation_data[sim_name]['net'] is not None]
```
Comment (ntolley) [inline hnn_core/gui/gui.py:1730]: I think I prefer `sim_names` here still, but otherwise this looks great!

**Review by ntolley — APPROVED** [review]
Thanks @kmilo9999! 

Is it possible to unit test this? Perhaps just check that there isn't loaded data in this field, I imagine it could be a oneliner added to an existing GUI test that loads data

**Comment by kmilo9999** [comment]
> Thanks @kmilo9999!
> 
> Is it possible to unit test this? Perhaps just check that there isn't loaded data in this field, I imagine it could be a oneliner added to an existing GUI test that loads data

Yes, I thought about it too. I'll work on it and add it to this PR

**Comment by kmilo9999** [comment]
@ntolley 
I added the unit test. One last review and I think you can merge it. I need it in order to continue working in the list of gui bugs.

File: hnn_core/gui/gui.py (hunk 1 above)
Comment (kmilo9999) [reply hnn_core/gui/gui.py]: done

**Pull request merged**